### DISCUSSION TITLE
SOLUCIONADO

### DISCUSSION THREAD

**Discussion by CarIosLopez** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#568] [Q&A]

**Comment by doguedogue** [comment]
es cuando no se proporcionó la descripción 😁

![image](https://user-images.githubusercontent.com/23409026/160966778-b7354b18-ffc1-44f7-85f7-9d3a5b57b7de.png)

**Reply by CarIosLopez** [reply]
Intente borrarlo porque apenas lo publique encontré la solución jaja, solo era que anote la ruta mal

**Reply by doguedogue** [reply]
genial, a veces eso pasa 🚀
